### PR TITLE
Generics power!

### PR DESCRIPTION
**Pull request by DmitriyTor**
More comfortable (don't look at the tests)

And you can save your custom codable struct

### CODE REVIEW

File: Sources/EKKeychainService/EKKeychainService.swift [hunk 1]
```diff
@@ -34,38 +36,51 @@ final public class EKKeychainService {
 
 extension EKKeychainService {
 
-    /// Persist the inputted object to keychain while assigning its key. 'String', 'Bool' & 'Data' object types are auto-handled. Other object types can NOT be stored within the keychain!
-    @discardableResult
-    public func set(_ object: Any?, withKey key: EKFieldKey) -> Bool {
+    /// Persist the inputted object to keychain while assigning its key.
+    /// 'String', 'Bool' & 'Data' object types are auto-handled.
+    /// Other object types will be convert to data with Codable func
+    public func set<T: Codable>(_ object: T?, withKey key: EKFieldKey) throws {
```
Comment (DmitriyTor) [inline Sources/EKKeychainService/EKKeychainService.swift:42]: holy shit, throws need to be removed

**Review by emvakar — CHANGES_REQUESTED** [review]
You need to supplement and correct the tests and then ill merge it

**Comment by DmitriyTor** [comment]
тесты разве не взлетели?

**Comment by emvakar** [comment]
> тесты разве не взлетели?

travis почему-то не запустил их и не чекнул

**Pull request merged**